File: 1476-count-negative-numbers-in-a-sorted-matrix/1476-count-negative-numbers-in-a-sorted-matrix.java
class Solution {
    public int countNegatives(int[][] grid) {
        int neg = 0;

        for(int[] arr:grid){
            Arrays.sort(arr);
            for(int num:arr){
                if(num < 0) neg++;
            }
        }
        
        return neg;
    }
}

File: 1476-count-negative-numbers-in-a-sorted-matrix/1476-count-negative-numbers-in-a-sorted-matrix.c
int countNegatives(int** grid, int gridSize, int* gridColSize) {
    int     neg = 0;
    for(int i = 0 ; i < gridSize ; i++){
        for(int j = 0 ; j < gridColSize[i] ; j++){
            if(grid[i][j] < 0)
            neg++;
        }
    }
    return neg;
    
}